### DISCUSSION TITLE
XXX马上迎来了他的第三个任期，在座各位的好日子还在后头捏

### DISCUSSION THREAD

**Discussion by ghost** [WeNeedHome/SummaryOfLoanSuspension#1084] [General]
RT, XXX迎来了他的第三个任期，在座各位的好日子还在后头。唐山烧烤店，铁链女，河南村镇银行，烂尾楼等等，接下来还有啥会发生在各位身上，那就让我们拭目以待吧。
![image](https://user-images.githubusercontent.com/110222065/182681025-bb171c84-d484-473a-a243-09e8c73aa13a.png)
![image](https://user-images.githubusercontent.com/110222065/182681100-acf95195-8142-4370-9e92-72114007bdcb.png)

**Comment by volans19** [comment]
这都是今年的，可能很多人都选择忘记了
卡车司机，上访老兵，p2p，红黄蓝，假疫苗，郑州水灾，无数冤魂野鬼，和等待在沉默中爆发的，找谁算总账呢，那就拭目以待吧

**Reply by wudilushi** [reply]
河南银行，郑州710

**Reply by TALK1949** [reply]
温州动车、汶川地震、天津大爆炸、北京清理低端人口

**Reply by itsoursduty** [reply]
最关键的33年前的没人说啊

**Reply by WlmAlex** [reply]
没人合并代码了啊，都16个PR了

**Comment by wedxz** [comment]
赵立坚 ： 那就拭目以待吧

**Comment by super-aviator** [comment]
貌似还提前改了宪法

**Comment by 1418478505** [comment]
打出习近平三个字很难吗兄弟

**Reply by super-aviator** [reply]
禁评

**Reply by CKwuliuqi** [reply]
说实话这个平台不需要VPN就能上，各位还是小心为好，发言不谨慎直接给肉身抓进去

**Comment by yellowcaicai** [comment]
习下李上，民心所向！！

**Reply by zxcs001** [reply]
指望另一个奴隶头子对牛马好一点，想多了吧

**Reply by yellowcaicai** [reply]
相比之下，比习强

**Reply by NultiluK** [reply]
大家捧场，老灯开奖

**Reply by CKwuliuqi** [reply]
小政府主义罢了，李上了不见得能有多好，但资本家的日子一定相当滋润

**Reply by lettermessenger** [reply]
没有任何一个青天大老爷手上是干净的

**Comment by ewkyrtas** [comment]
@MarkShawn2020  @MarkShawn2022    看 一下中天的 
 太老火了https://github.com/WeNeedHome/SummaryOfLoanSuspension/tree/main/images/%E8%B4%B5%E5%B7%9E%E7%9C%81/%E8%B4%B5%E9%98%B3%E5%B8%82/_%E4%B8%AD%E5%A4%A9%C2%B7%E5%90%BE%E4%B9%A1

**Reply by MarkShawn2020** [reply]
建议使用tg聊天

**Reply by winnoryder** [reply]
telegram不太安全

**Reply by HunterTOne** [reply]
> telegram不太安全

tg 为什么不安全？

**Reply by kajsnlk** [reply]
> > telegram不太安全
> 
> tg 为什么不安全？

tg注册也要手机号

**Comment by macleer** [comment]
张维为：中国人要自信

**Comment by Wilson403** [comment]
没想到不翻墙也可以看到这种敏感话题

**Reply by clleo888** [reply]
收紧言论, 批斗文化也是他的政绩呢!

**Comment by 1450-50w** [comment]
十年汇总:
一带一路:斯里兰卡破产，政府被推翻,巴基斯坦濒临破产
雄安新区:彻底烂尾失败
北京证交所:无疾而终
亚投行:大量减免国际债务，血本无归
千人计划:被当做间谍名单
芯片工程（10万亿)︰主要负责人全部落马
房地产:大面积烂尾楼断供
扶贫攻坚:6亿人口月入不到一千
节能减碳:为挽救经济不得不降暂停
厕所革命:大量废弃
香港:失去国际自由港和金融中心地位
台湾问题:和平统一已无可能
国际关系:欧美日印澳成为敌对关系

**Reply by CKwuliuqi** [reply]
最后一条我不认同，即便绥靖也迟早要被美搞的，只要中国人想过好日子就绕不开美国，所以这点无可厚非

**Reply by oceanzhang860411** [reply]
又來偷換概念，競爭和敵對分不清楚？

**Comment by Cegazhos** [comment]
写文案之前能不能动动脑子？怎么不把你爸暴扣你妈的屎篮子也甩上去呢？

希望梧桐的时候你有骨气站着

**Reply by ghost** [reply]
笑死你爹我了，纯纯的fw，爷看你在其他帖子也是这样骂，你妈逼的脑子是被屎吃了吗，真nm的脑瘫，吃nm的屎去吧。

**Reply by oceanzhang860411** [reply]
> 写文案之前能不能动动脑子？怎么不把你爸暴扣你妈的屎篮子也甩上去呢？
> 
> 希望梧桐的时候你有骨气站着

連武統都不敢打的太監，你也配姓趙？

**Reply by Cegazhos** [reply]
> > 写文案之前能不能动动脑子？怎么不把你爸暴扣你妈的屎篮子也甩上去呢？
> > 希望梧桐的时候你有骨气站着
> 
> 連武統都不敢打的太監，你也配姓趙？

hhh 果然台湾乡巴佬

**Reply by ghost** [reply]
> > > 写文案之前能不能动动脑子？怎么不把你爸暴扣你妈的屎篮子也甩上去呢？
> > > 希望梧桐的时候你有骨气站着
> > 
> > 
> > 連武統都不敢打的太監，你也配姓趙？
> 
> hhh 果然台湾乡巴佬

纯纯的牛马，社会的fw🤣。跟吃了nm的屎一样在这叫，你他娘笑死我。

**Reply by oceanzhang860411** [reply]
> > > 写文案之前能不能动动脑子？怎么不把你爸暴扣你妈的屎篮子也甩上去呢？
> > > 希望梧桐的时候你有骨气站着
> > 
> > 
> > 連武統都不敢打的太監，你也配姓趙？
> 
> hhh 果然台湾乡巴佬

你有資格說別人鄉巴佬？你有免費醫療嗎？你見過選票嗎？習包子經過你家小區你敢開窗？當奴隸還真是會上癮哦

**Reply by JanusTida** [reply]
还武统，每年都这么说，我作为大陆人都听到吐了。

**Reply by itsoursduty** [reply]
不会真有粉红觉得中共会武统吧....佩洛西的事情这么快忘了？美州长、日议员都访台了哦 你不会还在拭目以待吧

**Reply by Cegazhos** [reply]
> 不会真有粉红觉得中共会武统吧....佩洛西的事情这么快忘了？美州长、日议员都访台了哦 你不会还在拭目以待吧

相信武统的是粉红，那相信和平统一的又是啥？帽子都给你扣完了 还问啥呢？

**Comment by 505Games-oss** [comment]
二百斤:我看看我准备这个本本上有没有这个内容

**Reply by faceAndfaceandface** [reply]
听说有人患病了嘛

**Reply by xijinping0** [reply]
嘿嘿，小本本

[![小本本](https://img.youtube.com/vi/expxx4WNHgg/0.jpg)](https://youtu.be/expxx4WNHgg?t=63)

https://github.com/xijinping0
《[习近平与他的情人们](https://lovers.xijinping.one)》
